src/npr/rules_dorado/07_modbed.smk

```diff
@@ -29,7 +29,7 @@ rule bam2modbed:
     conda:
         "ont-ppp-modkit"
     shell:'''
-        if [[ "{params.do_modbed}" == "do_modbed" ]]; then
+        if [[ "{params.do_modbed,,}" == "yes" ]]; then
```

Comment:
extra commas?

Reply:
TIL they convert the content of the bash variable to lowercase (🗡️ it remains to be seen if it works in all the bashes out there, like other OSes), but it did work on CentOS 7 👼🏽